proposals/lip-0053.md

```diff
@@ -742,7 +742,7 @@ def execute(trs: Transaction) -> None:
     # Update the context to indicate that now we stop the CCM processing.
     ctx.ccmProcessing = False
 
-    # Update sidechain validators.
```

Comment:
Same as above.

Reply:
changed. Note that "sidechain" was wrong. It must be "mainchain" in this line.